### PR TITLE
feat: Update password reset pages UI

### PR DESCRIPTION
**Pull request by GoOz**
Used wizard common styles for password reset pages.

### CODE REVIEW

**Comment by GoOz** [comment]
[(𝐒) ETUO ma page de reset de mot de passe est personnalisée](https://trello.com/c/ONlJtSbF/1239-%F0%9D%90%92-etuo-ma-page-de-reset-de-mot-de-passe-est-personnalis%C3%A9e)

File: assets/templates/passphrase_renew.html [hunk 1]
```diff
@@ -5,7 +5,8 @@
     <title>{{.Title}}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="{{asset .Domain "/fonts/fonts.css"}}">
```
Comment (nono) [inline assets/templates/passphrase_renew.html]: I don't think `fonts.css` is still needed with Cozy-UI

File: assets/templates/passphrase_renew.html [hunk 2]
```diff
@@ -5,7 +5,8 @@
...
-    <link rel="stylesheet" href="{{asset .Domain "/styles/login.css"}}">
+    <link rel="stylesheet" href="{{asset .Domain "/styles/password-strength.css"}}">
```
Comment (nono) [inline assets/templates/passphrase_renew.html]: Can you add `.ContextName` to allow to override this CSS with dynamic assets?

File: assets/templates/passphrase_reset.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <title>{{.Title}}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="{{asset .Domain "/fonts/fonts.css"}}">
```
Comment (nono) [inline assets/templates/passphrase_reset.html]: Same here

**Pull request merged**